config_gen.py

```diff
@@ -303,6 +303,25 @@ def run(cmd, *args, **kwargs):
         print("\nRunning make...")
         run(make_args, env=env, **proc_opts)
 
+    elif(os.path.exists(os.path.join(project_dir, "Make/options"))):
+        print("Found OpenFOAM Make/options")
+        print("Preparing build directory...")
```

Comment:
The `Preparing build directory` line isn't needed, as we're no longer creating a temporary directory.